### PR TITLE
create settings from templates

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn ready for review

**Comment by miltontony** [comment]
@smn ready for review (and now with slack integration)

File: project/templates/configs/cms.settings.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+BROKER_URL = "redis://localhost:6379/0"
+CELERY_ALWAYS_EAGER = False
+CELERY_DEFAULT_QUEUE = "unicore_cms_django_{{app_type}}_{{country}}"
+CELERY_RESULT_BACKEND = "redis://localhost:6379/0"
+DEBUG = False
+GIT_REPO_URL = {{ git_repo_uri }}
+RAVEN_CONFIG = {'dsn': '{{raven_dsn_uri}}'}
+SSH_PRIVKEY_PATH = "/home/ubuntu/.ssh/django_cms"
+SSH_PUBKEY_PATH = "/home/ubuntu/.ssh/django_cms.pub"
```
Comment (smn) [inline project/templates/configs/cms.settings.py:48]: These aren't used by GitPython, it uses the process owners' SSH keys.

Comment (smn) [inline project/templates/configs/cms.settings.py:48]: I wonder how much this'll mess things up?

Comment (miltontony) [inline project/templates/configs/cms.settings.py:48]: True, but until we merge, we need to keep these

Comment (miltontony) [inline project/templates/configs/cms.settings.py:48]: @smn it shouldn't have any impact though - git should be able to handle it during the handshake process

Comment (smn) [inline project/templates/configs/cms.settings.py:48]: perhaps ... we'll find out in QA.

**Comment by smn** [comment]
:+1:

**Pull request merged**